### DISCUSSION TITLE
C API

### DISCUSSION THREAD

**Discussion by GrauBlitz** [thorvg/thorvg#1734] [Q&A] [answered]
Hello,

is there or will there be a C API for ThorVG? I am also fine with compiling the C++ version separately to use it with an C API in a C project if this is already an option.

Thanks!

**Comment by hermet** [comment, marked as answer]
@GrauBlitz hello, thorvg has it. https://github.com/thorvg/thorvg/blob/main/src/bindings/capi/thorvg_capi.h

if you are using the meson build system, you can enable the capi build with this:

```
$meson setup build -Dbindings="capi" ...
```

If you have any troubles, feel free to ask here. thanks!

**Reply by GrauBlitz** [reply]
Thank you very much! Seems like I was blind... :-)
